### PR TITLE
[SPARK-33908][CORE][FOLLOWUP] Correct Scaladoc of resolveDependencyPaths/resolveMavenDependencies

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### What changes were proposed in this pull request?
Fix un-correct doc of last change https://github.com/apache/spark/pull/30922#discussion_r551453193

### Why are the changes needed?
FIx doc

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
Builds finished correctly.

### CODE REVIEW

**Comment by AngersZhuuuu** [comment]
FYI @xkrogen @maropu

File: core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala [hunk 1]
```diff
@@ -1197,8 +1197,8 @@ private[spark] object SparkSubmitUtils {
    * Output a comma-delimited list of paths for the downloaded jars to be added to the classpath
```
Comment (xkrogen) [inline core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala:1197]: "Output a comma-delimited list" needs updating as well

**Review by xkrogen — COMMENTED** [review]
You missed one place in the Scaladoc.

Thanks for fixing this up!

File: core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala (hunk 1 above)
Comment (AngersZhuuuu) [reply core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala]: > "Output a comma-delimited list" needs updating as well

Emmmmm....,Updated ==

**Review by maropu — APPROVED** [review]

**Comment by maropu** [comment]
Pending GA.

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM. Thank you, @AngersZhuuuu and all.
Merged to master.

**Pull request closed without merging**